src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloud1WorkerCreatedFor2ExecutorsIT.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jenkins.plugins.computeengine.integration;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.jenkins.plugins.computeengine.ComputeEngineCloud;
+import com.google.jenkins.plugins.computeengine.client.ComputeClient;
+import hudson.model.labels.LabelAtom;
+import hudson.slaves.NodeProvisioner.PlannedNode;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Map;
+import java.util.logging.Logger;
+import java.util.logging.StreamHandler;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class ComputeEngineCloud1WorkerCreatedFor2ExecutorsIT {
+  private static Logger log =
+      Logger.getLogger(ComputeEngineCloud1WorkerCreatedFor2ExecutorsIT.class.getName());
+
+  private static final String MULTIPLE_NUM_EXECUTORS = "2";
+
+  @ClassRule public static JenkinsRule r = new JenkinsRule();
+
+  private static ByteArrayOutputStream logOutput = new ByteArrayOutputStream();
+  private static StreamHandler sh;
+  private static ComputeClient client;
+  private static Map<String, String> label =
+      ITUtil.getLabel(ComputeEngineCloud1WorkerCreatedFor2ExecutorsIT.class);
+
+  @BeforeClass
+  public static void init() throws Exception {
+    log.info("init");
+    ITUtil.initCredentials(r);
+    ITUtil.initCloud(r);
+    sh = ITUtil.initLogging(logOutput);
+    client = ITUtil.initClient(r, label, log);
+  }
+
+  @AfterClass
+  public static void teardown() throws IOException {
+    ITUtil.teardown(sh, logOutput, client, label, log);
+  }
+
+  @Test(timeout = 300000)
+  public void test1WorkerCreatedFor2Executors() {
+    ComputeEngineCloud cloud = (ComputeEngineCloud) r.jenkins.clouds.get(0);
+    cloud.addConfiguration(
+        ITUtil.instanceConfiguration(
+            ITUtil.DEB_JAVA_STARTUP_SCRIPT,
+            MULTIPLE_NUM_EXECUTORS,
+            ITUtil.LABEL,
+            label,
+            false,
+            false,
+            ITUtil.NULL_TEMPLATE));
+    // Add a new node
+    Collection<PlannedNode> planned = cloud.provision(new LabelAtom(ITUtil.LABEL), 2);
```

Comment:
I suppose this file could be used for node creation testing. Would it make sense to combine this and testWorkerCreated?